src/components/Input.js

```diff
@@ -3,13 +3,12 @@ import PropTypes from 'prop-types'
 
 const InputStyled = styled.input`
     width: 100%;
-    padding: 0 10px;
-    border-top-left-radius: 5px;
+    position: relative;
+    padding-left: 10px;
+    padding-right: 30%;
+    border-radius: 5px;
     border-bottom-left-radius: 5px;
```

Comment:
Why do you define a radius twice? Is there a specific reason?

Reply:
Whops, a left over 👍🏻